src/Packages/Microsoft.Internal.Extensions.DotNetApiDocs.Transport/Microsoft.Internal.Extensions.DotNetApiDocs.Transport.proj

```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+
+    <PropertyGroup>
+        <TargetFramework>$(LatestTargetFramework)</TargetFramework>
+        <TargetFrameworks>$(TargetFrameworks)</TargetFrameworks>
```

Comment:
Adding the `TargetFrameworks` property was my failed attempt to fix the error I'm getting:

```
C:\Program Files\dotnet\sdk\8.0.100-preview.7.23376.3\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDe
pendencyResolution.targets(266,5): error NETSDK1005: Assets file 'C:\Users\calope\source\repos\extension 
s\artifacts\obj\Microsoft.Internal.Extensions.DotNetApiDocs.Transport\project.assets.json' doesn't have  
a target for 'net8.0'. Ensure that restore has run and that you have included 'net8.0' in the TargetFram 
eworks for your project. [C:\Users\calope\source\repos\extensions\src\Packages\Microsoft.Internal.Extens 
ions.DotNetApiDocs.Transport\Microsoft.Internal.Extensions.DotNetApiDocs.Transport.proj::TargetFramework 
=net8.0]
```